mpt_extension_sdk/runtime/utils.py

```diff
@@ -105,3 +105,18 @@ def get_extension_variables(json_ext_variables):
 
         variables[var[0][4:]] = value
     return variables
+
+
+def get_api_url(extension):
```

Comment:
let's simplify it a little.

1. You check extension for None in both of the functions :-) I think one check is enough
2. Let's make get_api_urls function pure, return patterns and update urlpatterns outside of the function